import-scripts/importer.properties

```diff
@@ -0,0 +1,7 @@
+#comment
+#p1=hi
+p1=p1value
+p2=p2value
+p3=p3value
+ p4 = p4value     
```

Comment:
whitespace

Reply:
the whitespace was intentional ... this was a test file added to make sure that flanking whitespace was handled properly while reading property names. But I did not mean to check in this test file. I'll remove it.